src/main/java/com/thealgorithms/searches/agnosticBinarySearch.java

```diff
@@ -0,0 +1,32 @@
+package com.thealgorithms.searches;
+import java.util.*;
+public class agnosticBinarySearch {
+    public static int agnosticbinarySearch(int arr[],int key){
```

Comment:
```suggestion
public class sortOrderAgnosticBinarySearch {
    public static int find(int arr[], int key) {
```
The class name must say what it's agnostic to. The function name doesn't need to repeat what's already in the class name

Reply:
@siriak changes done can check it once

Reply:
@siriak changes will cleared can you check is there any modifications let me know